### ISSUE TITLE
Feature to extract block embeds during compile

### ISSUE THREAD

**Issue by djsudduth** [kevboh/longform#90]
Hello, I would like to request a feature step that would extract any block embed text within any of the draft's notes into the final compiled document. I tend to write with sections and sometimes my sections are block embeds into a new wrapper note with more writing. Then, when I need to compile a draft I want those to be extracted rather than block embeds in the final manuscript. Today, I have to manually copy-paste each embed in the final manuscript into the parent manuscript document.

Thanks!

**Comment by kevboh** [comment]
Makes sense! Thanks, I'll take a look, not in 2.0 but in a future release.

**Comment by djsudduth** [comment]
@kevboh any updates on this? I'm basically stuck not using longform without that extraction - sure would help!!

**Comment by kevboh** [comment]
@djsudduth I haven't started on this. If you're comfortable in code you should be able to do this with a custom step—if not, I'll be able to take a look soon, but I can't give you a precise date due to holiday obligations.

**Comment by djsudduth** [comment]
I did take a look at the code. Not sure how to step into block embeds to get that embed note data. I'll play around with it.

**Comment by kevboh** [comment]
Related: #121

**Comment by Servinjesus1** [comment]
[Related Plugin - Easy Bake](https://github.com/mgmeyers/obsidian-easy-bake)

Recursively "bakes" transclusions into a single file. Possibly a useful reference for integrating this functionality into a Longform compile step.

For simpler manuscripts, this could be a stand-in for Longform, but of course lacks the customizable script capabilities of a true Longform compile.

**Comment by djsudduth** [comment]
Any update on this yet? Would be a really useful addition.
